x-pack/plugins/enterprise_search/public/applications/app_search/components/result_settings/result_settings_table/non_text_fields_body.tsx

```diff
@@ -39,6 +40,10 @@ export const NonTextFieldsBody: React.FC = () => {
           </EuiTableRowCell>
           <EuiTableRowCellCheckbox>
             <EuiCheckbox
+              aria-label={i18n.translate(
```

Comment:
There were axe failures that I had missed on my previous PR. This helps a lot with the text reader, actually, so it doesn't just read "Raw, Raw, Raw".